avocado/core/html.py

```diff
@@ -115,7 +115,7 @@ def _get_sysinfo(self, sysinfo_file):
         return sysinfo_contents
 
     def hostname(self):
-        return self._get_sysinfo('hostname')
+        return self._get_sysinfo('hostname').strip()
```

Comment:
The thing is that, maybe, the trailing newline is indeed part of the *file* or *command* that was collected.  The hostname is extracted from a file, but we're not presenting it as the contents of a *file* or *command*... does it make sense?